apps/src/templates/CodeWorkspaceContainer.jsx

```diff
@@ -51,6 +57,52 @@ class CodeWorkspaceContainer extends React.Component {
       <div style={mainStyle} className="editor-column">
         <div id="codeWorkspace" style={styles.codeWorkspace}>
           {children}
+          <div id={MODAL_EDITOR_ID} style={styles[MODAL_EDITOR_ID]}>
+            <div
+              className="toolbar"
+              style={{
+                zIndex: 1,
+                position: 'absolute',
+                left: '142px',
+                top: '10px',
+                width: '800px',
+              }}
+            >
+              <div style={{float: 'right'}}>
+                <button type="button" id={MODAL_EDITOR_DELETE_ID}>
+                  delete
+                </button>
+                <button type="button" id={MODAL_EDITOR_CLOSE_ID}>
+                  close
```

Comment:
This is out of scope for this PR, but just wanted to make a note here - I see we use 'close' for the CDO version of function editor, but just wondering if 'save' would make more sense.

Reply:
Agreed. We will circle back on this when we get to ui polish.